.github/workflows/release.yml

```diff
@@ -5,28 +5,30 @@ on:
     types: [published]
 
 jobs:
+
   deploy:
+
     runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: bash -l {0}
-    container:
-      image: jammont/isofit:amd64
-      options: --user root
 
     steps:
-    - name: Enable ISOFIT Environment
-      run: |
-        echo PATH=/home/ray/anaconda3/envs/isofit/bin/:$PATH >> $GITHUB_ENV
-    - uses: actions/checkout@v2
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install build setuptools twine wheel
-    - name: Build and publish
-      env:
-        TWINE_USERNAME: __token__
-        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-      run: |
-        python -m build --sdist --wheel
-        twine upload dist/*
+      - uses: actions/checkout@v3
+
+      - name: Enable ISOFIT Environment
+        shell: bash
+        run: |
+          echo PATH=/home/ray/anaconda3/envs/isofit/bin/:$PATH >> $GITHUB_ENV
```

Comment:
This section should be deleted, right? This job is no longer executing in a Docker container.

Reply:
@pl-kevinwurster Good catch! This is of course not needed anymore. Removed in [5a8fb01](https://github.com/isofit/isofit/pull/450/commits/5a8fb01178892be15de73cb38b5a4967df93e197).